bin/polymer.js

```diff
@@ -30,6 +30,9 @@ resolve('polymer-cli', {basedir: process.cwd()}, function(error, path) {
   let args = process.argv.slice(2);
   let cli = new lib.PolymerCli(args);
   cli.run().catch((err) => {
+    if (err && err.constructor && err.constructor.name === 'CommandFailure') {
+      process.exit(err.exitCode);
```

Comment:
i know this exits as a side-effect, but adding a return here as well would be nice.